pylint/message/message_definition.py

```diff
@@ -70,11 +71,24 @@ def __repr__(self) -> str:
     def __str__(self) -> str:
         return f"{repr(self)}:\n{self.msg} {self.description}"
 
-    def may_be_emitted(self) -> bool:
-        """Return True if message may be emitted using the current interpreter."""
-        if self.minversion is not None and self.minversion > sys.version_info:
+    def may_be_emitted(
+        self,
+        py_version: tuple[int, ...] | sys._version_info | None = None,
```

Comment:
```suggestion
        py_version: tuple[int, ...] || None = None,
```

`sys._version_info` is equal to `tuple[int, ...]` right?

Reply:
I would have said so until very recently 😄  https://github.com/PyCQA/pylint/pull/7587#issuecomment-1272480072

Reply:
😄 

Never mind!